src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java

```diff
@@ -212,7 +212,7 @@ public Optional<NetworkModificationResult> moveModifications(UUID groupUuid, UUI
         List<ModificationInfos> movedModifications = networkModificationRepository.moveModifications(groupUuid, originGroupUuid, modificationsToMove, before)
             .stream()
             .filter(m -> !m.getStashed())
-            .map(ModificationEntity::toModificationInfos)
+            .map(networkModificationRepository::getModificationInfos)
```

Comment:
Change to be made also in saveAndApplyModifications method ?

Reply:
I didn't do it cause it was raising an error on the duplication of a tabular modification gathering generator modifications.

I still have the N+1 problem here (it's the only piece of code where I have it I think). Ok I'll fix it

Reply:
It is fixed